_data/sites.json

```
@@ -4396,6 +4396,17 @@
         ]
     },
 
+    {
+        "name": "Dictionary.com",
+        "url": "https://www.dictionary.com/account",
+        "difficulty": "easy",
+        "notes": "Account data will be deleted along with all the saved data and words. You fill find the detele button at the bottom of the page.",
+        "domains": [
+            "dictionary.com",
+            "dictionary.com/sign-in"
```

```suggestion
            "dictionary.com"
```
Second line was in the same domain, so it is not needed